test/disabled/test-sendfd.js

```diff
@@ -127,9 +126,9 @@ var srv = net.createServer(function(s) {
 srv.listen(SOCK_PATH);
 
 // Spawn a child running test/fixtures/recvfd.js
-var cp = child_process.spawn(process.argv[0],
-                             [path.join(common.fixturesDir, 'recvfd.js'),
-                              SOCK_PATH]);
+var cp = child_process.spawn(process.argv[0], [path.join(common.fixturesDir, 'recvfd.js'),
```

Comment:
Unrelated style change. Besides that, it will fail linting. Please don't ignore the friendly reminder to run `make -j4 test` you can see in the pull request template :)